client/components/domains/featured-domain-suggestions/index.jsx

```diff
@@ -0,0 +1,57 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { pickBy, includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import DomainRegistrationSuggestion from 'components/domains/domain-registration-suggestion';
+
+export class FeaturedDomainSuggestions extends Component {
```

Comment:
Curious - is this `export` on purpose for testing purposes? (IIRC, that's needed to get the actual component, not the wrapped one, but on the other hand I don't see tests that use this component)

Reply:
I tend to follow this pattern to enable testability in the future. You're right though - there are currently no tests using the exported component.

Reply:
>  I hope that in the future exposing the wrapped component like this won't be needed

Ditto that.